cryptarchia/cryptarchia.py

```diff
@@ -651,13 +662,13 @@ def phi(f: float, alpha: float) -> float:
 
     returns: the probability that this validator should win the slot lottery
     """
-    return 1 - (1 - f) ** alpha
+    return alpha * (- log(1-f) - 0.5 * alpha * log(1-f) ** 2)
 
 
-class MOCK_LEADER_VRF:
+class LEADER_VRF:
     """NOT SECURE: A mock VRF function"""
 
-    ORDER = 2**256
+    ORDER = 2**253
```

Comment:
where does this 253 come from again?

Reply:
Yeah true I mixed the Circom implementation and the risc0 one. I'll remove this. (In Circom it's needed for comparison purpose if we operate over a prime field of 255 bits.)